app/models/driver.rb

```diff
@@ -0,0 +1,49 @@
+require 'money'
+class Driver < ApplicationRecord
+  has_many :trips
+
+  validates :name, presence: true
+  validates :vin, presence: true
+
+  attribute :available, default: true
+
+  def mark_available
```

Comment:
Since this logic can also make a driver unavailable, is there a better name for this method other than `mark_available`?